spec/glueby/internal/wallet_spec.rb

```diff
@@ -16,6 +17,29 @@ def create_wallet; end
     it { should be_a Glueby::Internal::Wallet }
   end
 
+  describe 'load' do
+    subject { Glueby::Internal::Wallet.load(wallet_id) }
+
+    let(:wallet_id) { '0828d0ce8ff358cd0d7b19ac5c43c3bb' }
+
+    it { expect(subject).to be_kind_of Glueby::Internal::Wallet }
+
+    context 'if already loaded' do
+      let(:error) { Glueby::Internal::Wallet::Errors::WalletAlreadyLoaded }
+
+      it do
+        allow(Glueby::Internal::Wallet.wallet_adapter).to receive(:load_wallet).and_raise(error)
+        expect { subject }.not_to raise_error
```

Comment:
In this case, I think it's better to check that the wallet object with the specified wallet_id is returned, not that the exception does not occur.

Reply:
I fixed the spec so that it confirms wallet object returned by the method has same as specified wallet id 
a898cd1